bgpd/bgp_nexthop.h

```diff
@@ -30,7 +30,7 @@
 	(((nexthop_len) == 4 || (nexthop_len) == 12                            \
 		  ? AF_INET                                                    \
 		  : ((nexthop_len) == 16 || (nexthop_len) == 24                \
-				     || (nexthop_len) == 32                    \
+				     || (nexthop_len) == IPV4_MAX_BITLEN       \
```

Comment:
This is bytes.

Reply:
What do you mean?

Reply:
I mean this whole `NEXTHOP_FAMILY` macro checks for the number of bytes, not bits.
Even in the case you changed the return is AF_INET6, not AF_INET.
So if you want to change it, you should change 4 to IPV4_MAX_BYTES and 16 to IPV6_MAX_BYTES.
But I think it's better to leave it without change, as it looks weird to change only two of five numbers.

Reply:
Agree, reverting this.